app/controllers/provider_interface/application_choices_controller.rb

```diff
@@ -35,6 +35,31 @@ def show
 
     def notes
       redirect_to(action: :show) unless FeatureFlag.active?('notes')
+
+      @new_note_form = ProviderInterface::NewNoteForm.new
+      @notes = @application_choice.notes.order('created_at DESC')
+    end
+
+    def new_note
+      redirect_to(action: :show) unless FeatureFlag.active?('notes')
+
+      @new_note_form = ProviderInterface::NewNoteForm.new
+    end
+
+    def create_note
+      if FeatureFlag.active?('notes')
+        @new_note_form = ProviderInterface::NewNoteForm.new new_note_params
+
+        if @new_note_form.valid?
+          @new_note_form.save
```

Comment:
Might as well `!` if we're not looking at the return value

Reply:
Changed this to
```ruby
if @new_note_form.save
```
as #save includes a valid?